README.md

```diff
@@ -111,12 +111,19 @@ To build, you'll need a working [Haskell Stack install](https://docs.haskellstac
 
 To install for use with Atom and the command line, issue 'stack install' and then either add `$HOME/.local/bin` to your path, or symlink `$HOME/.local/bin/pact` somewhere in your PATH.
 
-Building with GHCJS
+Building with Nix / NixOS
 ---
 
-To build pact for browser use, add `--stack-yaml stack-ghcjs.yaml` to your stack commands. The script [web/build.sh](web/build.sh) builds Pact and minifies it; you'll need the [closure compiler](https://www.npmjs.com/package/google-closure-compiler) and a Java Runtime Environment installed.
+1. Go to https://nixos.org/nix/, click "Get Nix", follow the instructions to install the Nix package manager
+1. Edit `$NIX_CONF_DIR/nix.conf`
```

Comment:
the repeated 1s should be 2, 3, 4

Reply:
Oh, I thought markdown automatically converted lists like that into the correct numbers.  I guess not...

Reply:
oh wow! I had no idea about this! interesting. sorry for the misleading comment then!

Reply:
I don't think it does in github markdown.  I changed it back.